src/interfaces/SpokePool.ts

```diff
@@ -102,7 +102,6 @@ export interface V2Fill extends V2RelayData {
 
 export interface V3Fill extends V3RelayData {
   destinationChainId: number;
-  realizedLpFeePct: BigNumber;
```

Comment:
This doesn't get set by the spoke pool client. In fact, I don't think its possible to know this until the downstream client matches the fill with a deposit